graphdatascience/graph_data_science.py

```diff
@@ -48,12 +48,14 @@ def __init__(
             to a Neo4j AuraDS instance.
         database: Optional[str], default None
             The Neo4j database to query against.
+            # TODO whats off with the arrow part?
         arrow : Union[str, bool], default True
             Arrow connection information. This is either a bool or a string.
```

Comment:
```suggestion
            Arrow connection information. This is either a string or a bool.
```
Nitpick, to match the order in the Union type and in the list.